extensions/amp-user-notification/0.1/test/test-amp-user-notification.js

```diff
@@ -115,7 +115,7 @@ describes.realWin(
         'data-show-if-geo': 'nafta',
       });
       const impl = el.implementation_;
-      expect(impl.buildCallback.bind(impl)).to.not.throw;
+      expect(impl.buildCallback.bind(impl)).to.not.throw();
```

Comment:
Ah forgot that `.throw()` requires a wrapping function. These don't look right either.

Reply:
Bind returns a function, so this is fine.

Technically you don't need to pass a function if the test is expected `not` to throw. It'll still fail in the right cases :)